.azure/pipelines/components-e2e-tests.yml

```diff
@@ -54,6 +57,16 @@ jobs:
         searchFolder: '$(Build.SourcesDirectory)/src/Components/test/E2ETest/TestResults'
         testRunTitle: ComponentsE2E-$(AgentOsName)-$(BuildConfiguration)-xunit
       condition: always()
+    - task: PublishTestResults@2
+      displayName: Publish Quarantined E2E Test Results
+      inputs:
+        testResultsFormat: 'xUnit'
+        testResultsFiles: '*.xml'
+        searchFolder: '$(Build.SourcesDirectory)/artifacts/TestResults/$(_BuildConfig)/Quarantined'
+        testRunTitle: Quarantine-$(AgentOsName)-$(BuildConfiguration)-xunit
```

Comment:
```suggestion
        searchFolder: '$(Build.SourcesDirectory)/src/Components/test/E2ETest/TestResults'
        testRunTitle: ComponentsE2E-$(AgentOsName)-$(BuildConfiguration)-xunit
```

Reply:
Do **not** reuse the same `testRunTitle` as the previous `PublishTestResults@2` task. The test results should remain separate.

Reply:
Leaving as is then.